Post-WGLC feedback from Jari, WG feedback at IETF 126 (#87)
richsalz/draft-rsalz-2026bis: draft-ietf-procon-2026bis.md

      title: Guidance on Area Director Sponsoring of Documents
      target: https://datatracker.ietf.org/doc/statement-iesg-guidance-on-area-director-sponsoring-of-documents-20070320/
      date: "March 20, 2007"

>     RFCED:
>       title: RFC Editor
>       target: https://www.rfc-editor.org/

Reviewer: I don't think this is needed, see comments below
```suggestion
```

RFCs cover a wide range of
topics in addition to Internet Standards, from early discussion of
new research concepts to status memos about the Internet.

> For information about the input format, style guide, and
> supported output formats see
> [https://www.rfc-editor.org](https://www.rfc-editor.org).

Reviewer: This link is likely to become obsolete like the others. How about:

```suggestion
The RFC Editor maintains information about the input format,
style guide, and supported output formats.
```

and approval to become BCPs. See {{!RFC1796}} for further information.

> The full list of all RFCs, including the subseries, and lists organized

Reviewer: This is also too prescriptive about the current status of the website. I would just remove this sentence entirely.

recent version of the same specification, restarting the six-month
timeout period.

> The format of an Internet-Draft is mostly the same as for an RFC.
> Full details, including the naming conventions and required contents, can be
> found at {{RFCED}}.  Of particular importance is the legal boilerplate
> and copyright as described in the "Copyright Notice" section of
> [https://authors.ietf.org/en/required-content](https://authors.ietf.org/en/required-content).

Reviewer: ```suggestion
The format of an Internet-Draft is mostly the same as for an RFC.
The RFC Editor maintains the details of the format, such as naming
conventions and required contents. In particular, they maintain the
required legal boilerplate.
```
